### ISSUE TITLE
Unhappy pet sounds almost never play

### ISSUE THREAD

**Issue by ubertpendragon** [Regalis11/Barotrauma#11869]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

After browsing audio files we realized that nobody ever heard "unhappy" pet noises

Assuming this is the pet behavior across many pets:
`<petbehavior maxhappiness="100" happinessdecreaserate="0.016" hungerincreaserate="0.03" playforce="6.0">`
And pets get unhappy under <25 happiness

**It's almost impossible to make pets unhappy enough that they play their sounds. Pets will mostly die of starvation before getting unhappy.**

Hunger decreases faster than happiness, and food gives happiness. It takes a very long time for pets to be unhappy. Petting will give over 6 minutes of happiness too. 

Assuming you never pet the pets: _Graph by Vyk_

50 happiness 50 hunger (pet at spawn)
![image](https://user-images.githubusercontent.com/73229309/233107087-d3a4febf-e0fd-4ace-b1dd-e13df62050ad.png)
First reaches <25 at 1600 seconds, but shoots back up instantly after eating for another 600 seconds.

100 happiness 100 hunger
![image](https://user-images.githubusercontent.com/73229309/233107414-3095fb02-fff6-4e80-ae29-83ba8896f13e.png)
5300 seconds before the pet becomes unhappy

In short if you want to ever hear the unhappy audio, pets should probably get unhappy much faster than they currently are, and at least faster than hunger (0.045?)

### Version

v1.0.9.0

**Comment by Regalis11** [comment]
Addressed in https://github.com/Regalis11/Barotrauma-development/commit/87fb0f9dcc8d1411d010ef66b3564f1512835a8c:

> Some adjustments to fix pets practically never becoming unhappy. The happiness decrease rate was so low the pets were more likely to starve to death than to become unhappy. Changes I did to address this:
> 
> - Increased the happiness decrease rate (0.016 -> 0.035).
> - Instead of considering the pet unhappy below 20 happiness, made it a configurable setting and set it to 50. Makes more sense imo, because that's the point where the pet becomes too unhappy to produce items.
> - Play unhappy sounds also if the pet is hungry.

**Comment by Jasontti** [comment]
Tested.
No issues found.
Closing ticket.
